### ISSUE TITLE
Personally identifiable information (PII) in logs at INFO level

### ISSUE THREAD

**Issue by heidielliott72** [PennState/SCIMple-Identity#56]
INFO level logging in edu.psu.swe.scim.server.rest.BaseResourceTypeResourceImpl is sending the ScimUser to the logs (see processFilterAttributeExtensions method). This includes name, address, email and any extension data.

Since this type of data is considered PII should the log level be lower or removed altogether?

**Comment by smoyer64** [comment]
Yes it should be lower ... and some of the fields should probably be masked (incoming passwords, etc).  Let's use the "returned":"never" as a flag to indicate what shouldn't ever be shown (even on POST/PUT/PATCH).

@chrisharm @ussmith opinions?
